pkg/apps/application-list.jsx

```diff
@@ -98,11 +98,21 @@ export const ApplicationList = ({ metainfo_db, appProgress, appProgressTitle, ac
         comps.push(metainfo_db.components[id]);
     comps.sort((a, b) => a.name.localeCompare(b.name));
 
-    function get_config(name, distro_id, def) {
+    function get_config(name, os_release, def) {
+        // ID is a ingle value, ID_LIKE is a list
```

Comment:
"single"

Reply:
TIL that "ingle" is actually a word :fire: 

Fixed.